### DISCUSSION TITLE
如何在使用外部订阅地址的情况下增加自己的节点？

### DISCUSSION THREAD

**Discussion by puppetmm** [vernesong/OpenClash#2846] [Q&A]
我有使用订阅机场，但是偶尔也会出现不好用的情况，于是想添加一个自己的节点作为备份，但是各种测试，不是只能用机场，就是只能用自己的节点，一直没有弄明白如何共存，有没有成功的小伙伴？

**Comment by kingsey** [comment]
使用proxy provider只拉取机场节点，然后加上自己的，需要啃wiki自己写配置就是了
